app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -50,9 +50,9 @@ If you are already a qualified teacher, you do not need to retrain to teach in E
 * to pass criminal and professional safeguarding checks (these will be organised by your employer)
 * [a visa or immigration status allowing you to work in the UK](/non-uk-teachers/visas-for-non-uk-teachers)
 
-## Get an international relocation payment (IRP) worth £10,000
+## Get an international relocation payment (IRP) worth $nonuk_internationalrelocationpayment_value$
 
-If you're interested in moving to England to teach languages or physics in an English secondary school, you could be eligible for a single payment of £10,000 from the UK government. [Learn more about the international relocation payment](/non-uk-teachers/get-an-international-relocation-payment).
+If you're interested in moving to England to teach languages or physics in an English secondary school, you could be eligible for a single payment of $nonuk_internationalrelocationpayment_value$ from the UK government. [Learn more about the international relocation payment](/non-uk-teachers/get-an-international-relocation-payment).
```

Comment:
@Sarah-DfE Not sure `single payment` is correct now? Can you check with policy how this should be worded and raise a separate ticket and PR for this?